docsk8s/quick-start/sample-configuration-files.asciidoc

```diff
@@ -7,21 +7,33 @@ These configuration files are used in the <<ls-k8s-quick-start,{ls} and Kubernet
 
 You can download the files together as a link:https://github.com/elastic/logstash/blob/main/docsk8s/sample-files/logstash-k8s-qs.zip[zip archive].
 
+
+[[qs-setup-files]]
```

Comment:
I don't think that a free-floating anchor actually does anything. This content starts right below the previous heading, so you don't need an additional heading. I don't think you need this anchor either.